### ISSUE TITLE
Gene-reaction matrix in pymatbridge

### ISSUE THREAD

**Issue by Midnighter** [opencobra/cobrapy#1004]
@cdiener I have experienced the same issues as @beacamara regarding the `model_to_pymatbridge()`-function. I ran your suggested code:

```Sage
from pymatbridge import Matlab
mlab = Matlab()
mlab.set_variable("a", {"a": 1, "b": "hello"})
```

and that worked without any problems, so I inspected the function further. I discovered that the function itself creates an ordered dictionary with 17 key-value pairs. One of these keys is `rxnGeneMat` which is of type `sparse.dok.dok_matrix`. Within this sparse matrix, there are also key-value pairs, and all the keys are tuples of what I assume to be the position indexes of the sparse matrix. 

By converting `rxnGeneMat` into an array(), there are no longer key-value pairs, and it allows for the function to run. Would converting `rxnGeneMat` into an array be harmful to the purpose of the code in any way? I understand that for models with very large matrices this could pose a problem, but would this be a problem for smaller matrices as well?

_Originally posted by @molversmyr in https://github.com/opencobra/cobrapy/pull/909#issuecomment-693287854_

**Comment by cdiener** [comment]
@molversmyr Thanks that is good to know. I'll have to look at the code but a dok_matrix should be treatable as a normal numpy matrix (it's a [sparse matrix format from scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.dok_matrix.html)). But maybe something has changed in their API and this broke pymatbridge being able to send over sparse matrices...

**Comment by akaviaLab** [comment]
This should be closed since pymatbridge has been removed due to lack of maintainers. see #1197 which was merged as of release 0.25
